FhirToDataLake/deploy/templates/DeployAzureContainerApp.json

```diff
@@ -320,15 +340,31 @@
                 },
                 {
                     "name": "fhirServer__serverUrl",
-                    "value": "[parameters('fhirServerUrl')]"
+                    "value": "[if(variables('deployFhirToDatalake'), parameters('fhirServerUrl'), '')]"
                 },
                 {
                     "name": "fhirServer__version",
-                    "value": "[parameters('fhirVersion')]"
+                    "value": "[if(variables('deployFhirToDatalake'), parameters('fhirVersion'), '')]"
                 },
                 {
                     "name": "fhirServer__authentication",
-                    "value": "[if(parameters('authentication'), 'ManagedIdentity', 'None')]"
+                    "value": "[if(variables('deployFhirToDatalake'), parameters('serverAuthentication'), '')]"
+                },
+                {
+                    "name": "dataSource__type",
+                    "value": "[parameters('dataSourceType')]"
```

Comment:
nit: how about moving `datasource_type` to the front of `fhirServer__serverUrl`

Reply:
Sure, updated.